swatch-contracts/src/test/java/com/redhat/swatch/contract/service/ContractServiceTest.java

```diff
@@ -202,32 +198,6 @@ void createPartnerContract_UpdateContract() {
     assertEquals("New contract created", statusResponse.getMessage());
   }
 
-  @Test
-  void createPartnerContract_DuplicateContractThenDoNotPersist() {
-    PartnerEntitlementContract request = givenPartnerEntitlementContractRequest();
-    contractService.createPartnerContract(request);
-
-    StatusResponse statusResponse = contractService.createPartnerContract(request);
-    assertEquals("Existing contracts and subscriptions updated", statusResponse.getMessage());
-    verify(subscriptionRepository, times(2)).persist(any(Set.class));
-  }
-
-  @Test
-  void testCreatePartnerContractDuplicateBillingProviderIdNotPersist() {
```

Comment:
This test is no longer useful so deleting it. Duplicates are tested in other scenarios